library/alloc/src/borrow.rs

```diff
@@ -21,7 +21,6 @@ use Cow::*;
 impl<'a, B: ?Sized> Borrow<B> for Cow<'a, B>
```

Comment:
If you want it to look the same as the AsRef impl, you can also remove the `'a` lifetime:
```suggestion
impl<B: ?Sized> Borrow<B> for Cow<'_, B>
```

Reply:
In terms of how the rendered documentation comes out, I actually prefer the way it looks with the explicit lifetime, since otherwise it still renders as invalid Rust code with the `impl<'_>` part.